src/content/guide/tools-and-features/event-logs.md

```diff
@@ -0,0 +1,69 @@
+---
+word: Event Logs
+title: Event Logs
+order: 12 
+columns: two
+layout: guide.hbs
+---
+
+# {{title}}
+
+As your IoT fleet grows, the number of events sent by devices also increases. Having to browse through a big list of events is cumbersome. The new Event Logs page is tailored around improving this experience. It provides a handful of tools, like filtering, modifiers which enable you to narrow down your search for events, making it easier to view only the data that is relevant to you. Because the page was developed with scalability in mind it is now able to handle hundreds of events coming from your devices at the same time.
+
+To visit the new page go to [https://console.particle.io/events](https://console.particle.io/events)
+
+<img src="/assets/images/eventlogs/full.png" class="full-width" />
+
+### Filtering the events
+
+Filters are a great way to narrow down your search and only see events that are relevant.
+
+You can filter the events by writing anything in the input. Your query will be compared with the event name, data, publisher, and date.
+<img src="/assets/images/eventlogs/filter_word.png" class="full-width" />
+
+#### Modifiers
+
+Besides writing in the input, you can use modifiers to narrow down your search even further.
+
+<img src="/assets/images/eventlogs/filters.png" class="full-width" />
+
+- `product` Filter by product (example: `product:my-product`)
```

Comment:
I thought the `product` filter was going away. It doesn't make sense to me: if I'm on a user or individual device page, filtering by product doesn't make sense. If I'm on a product page, it should always filter by product anyway.